### PR TITLE
Chat: Improve results tables

### LINKED ISSUE #778: Chat: improve result tables
That's how the tables currently look like:

Improvements:

- [x] Make the tables sortable (use material UI's data table again?)
- [x] Proportions should be formatted as percentages with two decimal places
- [ ] Check the UI/UX on mobile and improve if needed
- [x] Add a subtle download icon to download the data as a CSV

### PR DESCRIPTION
**Pull request by JonasKellerer**
Resolves #778

### CODE REVIEW

**Comment by JonasKellerer** [comment]
'Check the UI/UX on mobile and improve if needed' is still open. I checked it through the browser tools, where it seems okay. I think not needing to scroll due to the pages helps. Feedback is very welcome.

File: src/components/chat/IncomingResponseMessage.tsx [hunk 1]
```diff
@@ -27,30 +27,87 @@ export const IncomingResponseMessage = ({
...
+        columns={headers}
+        rows={rows}
+        autoHeight={true}
+        paginationModel={paginationModel}
+        pageSizeOptions={[5, 10, 20]}
+        onPaginationModelChange={model => setPaginationModel(model)}
+        sx={{
+          backgroundColor: 'background.paper',
+        }}
+        slots={{ toolbar: CustomToolbar }}
```
Comment (chaoran-chen) [inline src/components/chat/IncomingResponseMessage.tsx]: I think that it would be better to save some space and make the rows as narrow as possible

**Review by chaoran-chen — CHANGES_REQUESTED** [review]
Thank you very much! Only one small request (see below), otherwise it's perfect!

**Comment by chaoran-chen** [comment]
Oh, and I don't think that we need the "print" option. Let's only offer the CSV download for now.

**Review by chaoran-chen — APPROVED** [review]

**Pull request merged**